beamer/tests/agent/unit/utils.py

```diff
@@ -136,6 +137,13 @@ def make_claim_challenged(
 def make_context() -> Tuple[Context, Config]:
     checker = TokenChecker([])
 
+    target_descriptor = search(id=TARGET_CHAIN_ID)
+    source_descriptor = search(id=SOURCE_CHAIN_ID)
+    if len(target_descriptor) == 0:
+        register(TARGET_CHAIN_ID, ChainDescriptor(TARGET_CHAIN_ID, "ethereum", "testnet"))
```

Comment:
Checking `len(target_descriptor) == 0` is not really idiomatic Python, we should rather use `not ... ` here, e.g.
```python
    if not search(id=TARGET_CHAIN_ID):
        register(TARGET_CHAIN_ID, ChainDescriptor(TARGET_CHAIN_ID, "ethereum", "testnet"))
```